omego/fileutils.py

```diff
@@ -42,14 +43,26 @@ def update(self, current):
                 self.marker * self.n, p, current, self.total)
 
 
-def open_url(url, httpuser=None, httppassword=None):
+def open_url(url, httpuser=None, httppassword=None, method=None):
     """
     Open a URL using an opener that will simulate a browser user-agent
     url: The URL
     httpuser, httppassword: HTTP authentication credentials (either both or
       neither must be provided)
+    method: The HTTP method
+
+    Caller is reponsible for calling close() on the returned object
     """
-    opener = urllib2.build_opener()
+    if os.getenv('OMEGO_SSL_NO_VERIFY') == '1':
+        # This needs to come first to override the default HTTPS handler
+        log.debug('OMEGO_SSL_NO_VERIFY=1')
+        sslctx = ssl.create_default_context()
```

Comment:
Is that the full error message? Did `create_default_context` fail, or does it not exist?